training/src/anemoi/training/config/interpolator.yaml

```diff
@@ -29,9 +29,7 @@ model:
   grid_skip: 0 # Which of the input indices to use as residual connection, null if none.
 
 training:
-  model_task: anemoi.training.train.forecaster.GraphInterpolator
-
-# Instead of inferred using multistep and timeincrement, specify time indices explicitly
+  # Instead of inferred using multistep and timeincrement, specify time indices explicitly
   explicit_times:
```

Comment:
Was just wondering why we need to put the explicit times in again as it is already in the default file?